### DISCUSSION TITLE
How to removed a light switch from the plug in

### DISCUSSION THREAD

**Discussion by luowilliam326** [plasticrake/homebridge-tplink-smarthome#352] [Q&A]
After changing out a switch that was broken, the new switch is added automatically to homebridge and home app which is great. My question is how do I get rid of the old switch from the plug in? I deleted it from Kasa app but it seems the plugin is still keeping it.

**Comment by jasonlboggs** [comment]
You need to find it in your cached accessories, in homebridge, and remove it
![image](https://github.com/user-attachments/assets/1c44427d-490e-4e2f-ba17-be85bc1b1242)
